tests/console/validate_activation_encrypted_partition.pm

```diff
@@ -29,6 +29,7 @@ sub run {
     verify_cryptsetup_message($test_data->{device_status}->{message}, $status->{message});
     verify_cryptsetup_properties($test_data->{device_status}->{properties}, $status->{properties});
     select_console 'installation';
+    set_var('AY_INTERRUPT', '0') if get_var('AY_INTERRUPT');
```

Comment:
it would only work for 1 interruption, one check, after that will go on. Also that variable should be documented.

Reply:
> it would only work for 1 interruption, one check, after that will go on. 

Well yes, that is the goal, what is you concern here ?

Reply:
that other checks might be required in future in this point so it only interrupt for one module, two module cannot be schedule in the middle, that is what doesn't make it general.

Reply:
But if we want to execute several modules we can just put this one in last, no ?